packages/salesforcedx-vscode-core/src/orgPicker/orgList.ts

```diff
@@ -88,31 +92,28 @@ export class OrgList implements vscode.Disposable {
         // scratch orgs parented by other (non-default) devHub orgs
         continue;
       }
-      const isExpired =
-        authFields && authFields.expirationDate
-          ? today >= new Date(authFields.expirationDate)
-          : false;
+      const isExpired = authFields?.expirationDate
+        ? today >= new Date(authFields.expirationDate)
+        : false;
 
+      if (isExpired) {
+        // If the scratch org is expired we don't want to see it in the org picker
+        continue;
+      }
       const aliases = await ConfigUtil.getAllAliasesFor(orgAuth.username);
-      let authListItem =
-        aliases && aliases.length > 0
+      const authListItem =
+        aliases?.length > 0
           ? `${aliases.join(',')} - ${orgAuth.username}`
           : orgAuth.username;
 
-      if (isExpired) {
-        authListItem += ` - ${nls.localize(
-          'org_expired'
-        )} ${String.fromCodePoint(0x274c)}`; // cross-mark
-      }
-
       authList.push(authListItem);
     }
     return authList;
   }
 
   public async updateOrgList(): Promise<string[]> {
     const orgAuthorizations = await this.getOrgAuthorizations();
-    if (orgAuthorizations && orgAuthorizations.length === 0) {
+    if (orgAuthorizations?.length === 0) {
```

Comment:
I know this is existing code, but I don't see how getOrgAuthorizations() returns an undefined and why are we checking for an empty array when the filterAuthInfo will do the same?